app/src/app/components/contents/downloads/Downloads.tsx

```diff
@@ -0,0 +1,67 @@
+import { useParams } from "react-router-dom";
+import { usePacketOutletContext } from "../../main/PacketOutlet";
+import { PacketHeader } from "../packets";
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from "../../Base/Accordion";
+import { Roles } from "../../../../types";
+import Artefacts from "./Artefacts";
+import OtherFiles from "./OtherFiles";
+import { Card, CardContent } from "../../Base/Card";
+import FileRow from "./FileRow";
+
+export default function Downloads() {
+  const { packetId, packetName } = useParams();
+  const { packet } = usePacketOutletContext();
+  const artefacts = packet?.custom?.orderly.artefacts;
+  const inputs = packet?.custom?.orderly.role
+    .filter((input) => [Roles.Resource, Roles.Shared].includes(input.role));
+  const packetIsFromOrderly = !!packet?.custom?.orderly;
+
+  return (
+    <>
+      <PacketHeader packetName={packetName ?? ""} packetId={packetId ?? ""} displayName={packet?.displayName} />
+      {packet && (<>
+        {!packetIsFromOrderly ? (<>
+          <h3>Downloads</h3>
+          <Card>
+            <CardContent className="p-0">
+              <ul>
+                {packet.files.map((file, index) => (<li key={index}>
+                  <FileRow path={file.path} packet={packet} />
+                </li>))}
+              </ul>
+            </CardContent>
+          </Card>
+        </>) : (<>
+          {!!artefacts?.length && !!inputs?.length ? (
```

Comment:
these nested ternary operators are hard to read and understand. Maybe best to extract into another component and use `If statements`

Reply:
if ya feel ya drilling too deep, feel free to call the context again from the nested child component and not pass as props... thats what the context provides us; not needing to drill too deep.... Its accessing the same state so new "request" is being made

Reply:
OK great, thanks